LocalFeeder/FeederSimulator.py

```diff
@@ -296,12 +299,14 @@ def load_feeder(self):
         self._pvsystems = set()
         for PV in get_pvsystems(dss):
             self._pvsystems.add("PVSystem." + PV["name"])
+
+        if self.tap_setting is not None:
+            dss.Text.Command(f"batchedit transformer..* wdg=2 tap={self.tap_setting}")
```

Comment:
why have you hard-coded to winding 2? Also, it is really needed instead of use magic number use a variable that folks reviewing can understand

Reply:
This is what I saw in the original GO-Solar code used for testing. I really don't know why this is here. Should it be? @yangrui05

Reply:
I've changed it after testing. I'm not sure what effect it had. The plots are the same on ieee123.

Reply:
Depending on the tyee of regulator, you may want to change tap on the primary side